### PR TITLE
Exit Rari Fuse #18, Enter Convex Iron Bank

### PR DESCRIPTION
**Pull request by green-jeff**
Exit Rari Fuse #18, enter curve-convex Iron Bank (Yearn lending protocol) This will provide theoretical APY of 6.05% APY / 4.85% for end user

### CODE REVIEW

**Comment by patidarmanoj10** [comment]
I would recommend exit pool 18 and move to Convex4 FRAXPool. That strategy is ready and getting good APY.

**Review by jgarzik — APPROVED** [review]

File: mainnet/vaDAI.md [hunk 1]
```diff
@@ -3,10 +3,11 @@
 |-------: | --------|
 |Convex | 0%     |
 |Convex4 FRAXPool| 23% |
+|Convex4 IronBank| 20% |
```
Comment (patidarmanoj10) [inline mainnet/vaDAI.md:6]:
```suggestion
|Convex4 FRAXPool| 43% |
|Convex4 IronBank| 0% |
```

**Review by patidarmanoj10 — CHANGES_REQUESTED** [review]
Instead of adding new convex, should increase FRAX pool convex

**Review by jcvernaleo — CHANGES_REQUESTED** [review]
After PR #68 from @patidarmanoj10 this one now conflicts.  Could you either update it to only have the changes that don't touch vaDAI or close it and make a new one with the changes that are left?

**Comment by patidarmanoj10** [comment]
Outdated PR. will raise new PR if we want to update weight.

**Pull request closed without merging**